### ISSUE TITLE
Could not load signature of OpenBveApi.Hosts.HostInterface:RegisterTexture due to: ...

### ISSUE THREAD

**Issue by edgreenberg** [leezer3/OpenBVE#873]
### Description

OpenBve 1.9.0.0 on newly Installed Ubuntu 22.10.  Downloaded as a zip file from ```https://openbve-project.net/download/``` not a package.  Installed Mono 6.8.0.105 and all it's dependencies.  When running OpenBve with ```mono OpenBve.exe``` I get:

```
Could not load signature of OpenBveApi.Hosts.HostInterface:RegisterTexture due to: Could not load file or assembly 'System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.

Unhandled Exception:
System.TypeLoadException: VTable setup of type OpenBve.Host failed
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: VTable setup of type OpenBve.Host failed
```

### Reproduction

unzip, cd into directory, and run ```mono OPenBve.exe```

## Route

Didn't get that far

## Train

Didn't get that far

## Logs

Didn't get that far
### Related information 

* Ubuntu 22.10
* No controls -- Didn't get that far

**Comment by leezer3** [comment]
You're missing the System.Drawing assembly from Mono.

Specificially, this package (if using the Ubuntu repositories):
https://packages.ubuntu.com/kinetic/amd64/libmono-system-drawing4.0-cil

Not sure off the top of my head what installing Mono adds by default, but this is the full list of Mono libraries we use:
```
mono-runtime
libmono-corlib4.5-cil
libmono-system-drawing4.0-cil
libmono-system-windows-forms4.0-cil
libmono-system4.0-cil
libmono-system-xml-linq4.0-cil
libmono-i18n4.0-all
libmono-microsoft-csharp4.0-cil
```

You might consider trying the .deb package. as this will handle the full list of requirements.

Also, note that I've just pushed a small hotfix to 1.9.0.1 :)

**Comment by edgreenberg** [comment]
Many thanks for the quick response. I will attempt to resolve things with
suggestions you made. I want to mention that when I try to install the .deb
file I got an error. It wasn't specific at all, and there was nothing in
the /var/log/apt to explain it. That's why I fell back to the zip file. I
will post a separate issue regarding that error. Unfortunately I've started
work, so I probably won't play with this again until tonight.

Thanks again for all the hard work you do to bring us OpenBVE, so I can get
my Subway fix :-)
